docs/reference/supported-environments.md

```diff
@@ -33,52 +33,108 @@ For example, 1.2+ means support for the minor version 2, and any higher minors (
 
 ## Camunda 8 Self-Managed
 
-We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Please follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
+### Minimum requirements
+
+The following are the minimum supported requirements for cluster specification and dependencies:
+
+- 4vCPUs, 15 GB Memory
+- 4 nodes
+- OpenJDK 17+
+- Elasticsearch 8.9+ or Amazon OpenSearch 2.5+
+- PostgreSQL 14.x or Amazon Aurora PostgreSQL 14.x
+- ...
+
+### Recommendations
```

Comment:
Should we add a section for none-Kubernetes deployments? 
We still support Docker standalone (Compose) and Manual, which could be deployed on IaaS Cloud, VMs or Bare-metal.

Reply:
My understanding is that Docker Compose is not for production use, and I would not highlight it here. This should be production only. Please confirm you agree with the split [here](https://docs.camunda.io/docs/self-managed/platform-deployment/overview/#deployment-recommendation).

We've previously been very explicit about Helm as our recommended deployment path. Should we remove this and promote all deployment methods? This would be a change in direction in how we've presented this content.

Reply:
We officially support [4 production configurations](https://docs.camunda.io/docs/self-managed/platform-deployment/overview/#production) 
Let me clarify that we don't support docker-compose 'script' that deploys C8, but we support C8 deployed with Docker images (Which in reality would be done with customer-provided scripts using docker-compose similar to customer providing script to deploy JAR's)
We highly recommend HELM and Kubernetes, but our customers deploy with other options as well and we support them.